### ISSUE TITLE
[1.10] Inability to add Marble from Project Red to Chisel-able blocks in the "marble" group

### ISSUE THREAD

**Issue by AlexanderBulanov** [Chisel-Team/Chisel#466]
I used ModTweaker's handler /mt chiselGroups and found no "marble" block group among the generated ones in the minetweaker.log output. Does such group not exist in Chisel, or does it have a different name?

**Comment by AlexanderBulanov** [comment]
After updating Chisel to the new version I no longer have this issue. When using the same command /mt ChiselGroups, I now got 133 Chisel Groups back, with "marble" being the extra one I didn't get with the old version. Will report back if any complications arise.

**Comment by AlexanderBulanov** [comment]
It seems like the issue wasn't fully resolved. I can either log into the world with Craft/Modtweaker not functioning at all, or have the world be stuck in infinite loop trying to load itself. Here's the Pastebin of the log I received once I terminated my instance and stopped the infinite world-loading loop -
 https://pastebin.com/TyEfeVN8. In addition, here are the Craft/Modtweaker commands I used:

mods.chisel.Groups.addVariation("marble", <<projectred-exploration:stone>>);
mods.chisel.Groups.addVariation("marble", <<projectred-exploration:stone:1>>);
